docs/framework.md

```diff
@@ -1151,124 +1116,26 @@ Information on the eligibility criteria for participants in the pre-qualificatio
-
-**Example**: See [section I.4](#i-4-project-economic-and-social-benefits) for JSON and flattened examples of the `documents` building block.
-
-### II.2. List of pre-qualified suppliers
-
-Pre-qualification or shortlist
-
-Details of the bidders should be provided using [Organization](https://standard.open-contracting.org/1.1/en/schema/reference/#parties) objects in the [parties](../reference/schema/#release-schema.json,,parties) array.
-
-* If the bidder is shortlisted or invited to submit a proposal following the pre-qualification process, the party's `roles` array should include the 'qualifiedBidder' code.
-* If the bidder is **neither** shortlisted **nor** invited to submit a proposal following the pre-qualification process, the party's `roles` array should include the 'disqualifiedBidder' code.
-
-**Example**: See [section I.14.2](#i-14-2-organization-details) for JSON and flattened examples of the `organization` building block.
-
-### II.3. Tender
-
-> The process by which bids are invited from interested parties to carry out the project. ([Source](https://ppp-certification.com/ppp-certification-guide/glossary))
-
-#### II.3.1. Dates - Tender period
-
-<div class="disclosure-timing">
-
-```eval_rst
-==  ==  ==  ==  ==  ==
-PP  P   A   CC  I   R
-==  ==  ==  ==  ==  ==
-P   U                 
-==  ==  ==  ==  ==  ==
-
-```
-
-</div>
-
-The `tender/tenderPeriod` field should be used to provide the period during which the tender is open for submissions, `tenderPeriod/endDate` should contain the closing date for tender submissions.
-
-**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/tenderPeriod](../reference/schema/#release-schema.json,,tender/tenderPeriod) field.
-
-**JSON example:**
-
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/1/tender/tenderPeriod
-
-```
-
-**Flattened example** (showing top-level fields only)
-
-<div class="flattened-example">
-
-```eval_rst
-.. jsoninclude-flat:: examples/full.json
-  :jsonpointer: /releases/1/tender/tenderPeriod
   :ignore_path: /releases/1/
 ```
 
 </div>
 
-#### II.3.2. Dates - Enquiry period
-
-<div class="disclosure-timing">
-
-```eval_rst
-==  ==  ==  ==  ==  ==
-PP  P   A   CC  I   R
-==  ==  ==  ==  ==  ==
-P   U                 
-==  ==  ==  ==  ==  ==
-
-```
-
-</div>
-
-The `tender/enquiryPeriod`field should be used to provide the period during which enquiries may be made and answered.
+#### II.1.6. Request for Qualification
 
-**Schema**: Information can be provided using a [Period](https://standard.open-contracting.org/1.1/en/schema/reference/#period) object in the [tender/enquiryPeriod](../reference/schema/#release-schema.json,,tender/enquiryPeriod) field.
-
-**JSON example:**
-
-```eval_rst
-.. jsoninclude:: examples/full.json
-  :jsonpointer: /releases/1/tender/enquiryPeriod
+Request for Qualification (RFQ):
```

Comment:
Do we need this line, which repeats the heading?

Reply:
It's in keeping with the pattern used in the other headings: provide the name used in the framework under the OCDS for PPPs heading. Although, in this case, the two are identical. Happy to remove it if you prefer?

Reply:
I suggest either removing it or putting it inside the following blockquote, so that it's clear it is being quoted from the framework.

Reply:
Removed in https://github.com/open-contracting-extensions/public-private-partnerships/pull/245/commits/ef0651de03805ab5f86c3493e16fb4452b0422ae